### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Ahmed-Ashraf-Marzouk**
Reformatted the README to improve readability.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,20 +5,26 @@
 
 CIDAR: Culturally-Relevant Instruction Dataset For Arabic. CIDAR contains 10,000 instructions and their outputs. The dataset was created by selecting around 9.2k samples from [Alpagasus](https://huggingface.co/datasets/mlabonne/alpagasus) dataset then translating it using ChatGPT. In addition, we append that with around 800 instructions from the webiste [Ask the teacher](https://learning.aljazeera.net/ar/asktheteacher). All the 10,000 samples are reviewed by around 10 reviewers. 
 
-## CIDAR Annotation
-First install the requirements `pip install -r requirements.txt` then run the app using flask `flask --app app run `.
+## 1. CIDAR Annotation
+### To run the app:
+1. Install the requirements `pip install -r requirements.txt`.
+1. Run the app using flask `flask --app app run `.
```
Comment (zaidalyafeai) [inline README.md]: should be 1. and 2. ?

File: README.md [hunk 2]
```diff
@@ -5,20 +5,26 @@
...
+## 2. Loading the dataset
+You can download the dataset from [HuggingFace - CIDAR](https://huggingface.co/datasets/arbml/CIDAR). 
 
 ```python
 from datasets import load_dataset
 cidar = load_dataset('arbml/CIDAR')
 ```
 
-Example sample from the dataset:
+## 3. Sample from the dataset:
```
Comment (zaidalyafeai) [inline README.md:38]: Maybe explain what are the contents of the example, instruction, output, etc.

**Review by zaidalyafeai — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Ahmed-Ashraf-Marzouk) [reply README.md]: Yes, you are right thanks.

File: README.md (hunk 2 above)
Comment (Ahmed-Ashraf-Marzouk) [reply README.md:38]: I will give it a try Insha'Allah.

File: README.md (hunk 1 above)
Comment (Ahmed-Ashraf-Marzouk) [reply README.md]: It seems to work both ways, but I will change it is less readable.

**Review by Ahmed-Ashraf-Marzouk — COMMENTED** [review]
I have made, mostly, the updates required.

**Review by zaidalyafeai — APPROVED** [review]

**Pull request merged**